app/assets/stylesheets/home.scss

```diff
@@ -1,9 +1,38 @@
 body {
-  background: image_url("polygon_green.jpg") no-repeat center center fixed;
-  background-size: cover;
-  background-position: center top;
-  overflow-x: hidden;
-  color: white;
-  font-family: Bandung, Arial, sans-serif;
-  font-weight: 400;
+  background-color: #bdc3c7;
+  font-family: 'Lato', sans-serif;
+}
+
+#featured {
+  background-color: #1abc9c;
+  height: 388px;
+  width: 100%;
+}
+
+#featured-small {
+  background-color: #1abc9c;
```

Comment:
Color literals like `#1abc9c` should only be used in variable declarations; they should be referred to via variable everywhere else.